test/functional/DDR_Test/src/j9vm/test/ddrext/junit/TestThread.java

```diff
@@ -238,48 +223,38 @@ private String getStackSlotsValues(String stackslotOutput) {
 		return results;
 	}
 
-	/*
-	 * If needELS, we return the second param list for stackslots:
-	 * thread,sp,a0,pc,literals,elsElse,we return the first param list for
-	 * stackslots : thread,sp,a0,pc,literals
+	/**
+	 * Constructs the parameter list of stackslots (thread, sp, a0, pc, literals) from the output.
+	 * @param stackOutput the output of the stackslots command
+	 * @param threadAddress the address of the walked thread
+	 * @return the parameter list for stackslots, or null if it failed to construct the list
 	 */
-	private String getStackSlotsParamSet(String stackOutput,
-			String threadAddress, boolean needELS) {
-		String output[] = stackOutput.split(Constants.NL);
+	private static String getStackSlotsParamSet(String stackOutput, String threadAddress) {
+		String[] output = stackOutput.split(Constants.NL);
 		String paramList = threadAddress;
 		for (String aLine : output) {
 			if (aLine.contains("Initial values")) {
-				String sp = null, pc = null, literals = null, a0 = null, els = null;
-				String tokens[] = aLine.split(",");
-				for (int i = 0; i < tokens.length; i++) {
-					String token = tokens[i].trim();
+				String sp = null;
+				String pc = null;
+				String literals = null;
+				String a0 = null;
+				String[] tokens = aLine.split(",");
+				for (String token : tokens) {
+					token = token.trim();
 					if (token.contains("walkSP")) {
-						sp = tokens[i].split("=")[1].trim();
-					} else if (token.startsWith("PC")) {
+						sp = token.split("=")[1].trim();
+					} else if (token.contains("PC")) {
 						pc = token.split("=")[1].trim();
-					} else if (token.startsWith("literals")) {
+					} else if (token.contains("literals")) {
 						literals = token.split("=")[1].trim();
-					} else if (token.startsWith("A0")) {
+					} else if (token.contains("A0")) {
```

Comment:
Only the first key (`walkSP`) needs to use `contains()`, please don't change the others.

Reply:
Sorry, fixed now.